cinema/models.py

```diff
@@ -35,12 +39,19 @@
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> pathlib.Path:
+    ext = pathlib.Path(filename).suffix
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}{ext}"
+    return pathlib.Path("upload/movies") / filename
```

Comment:
The `movie_image_path` function returns a `pathlib.Path` object, but Django's `ImageField` expects a string path. Consider converting the `pathlib.Path` object to a string using `str()` before returning it.